README.md

```diff
@@ -15,51 +16,51 @@ We welcome issues, feature requests, and contributions to the [repo](https://git
 
 Create outlines:
 
-![outlines](img/screenshots/01_outlines.gif)
+![outlines](images/screenshots/01_outlines.gif)
```

Comment:
Is it necessary?
I found folder names like "src", "img", "js" quite standard. And, if we rename "img" into "images" why not renaming "src" into "sources", "docs" into "documents" ?

I vote to keep "img".

Reply:
We had both. `img` and `images`.
I tend to prefer the former.

For some reason I tend to like `src`, `images` and `documention`. But that personal taste